### PR TITLE
Bump kotlin version and coroutineworker version

### PR DESCRIPTION
**Pull request by DanielRBaird**
No migration changes for once. Nice.

### CODE REVIEW

**Comment by DanielRBaird** [comment]
Oof. Looks like we need a newer Xcode version to be used to build.

**Comment by benasher44** [comment]
Hmm weird. I'll look more today

**Comment by DanielRBaird** [comment]
I thought this was going to be simple, but Jesse Squires suggestions don't seem to be working for me. https://www.jessesquires.com/blog/2020/01/06/selecting-an-xcode-version-on-github-ci/

**Comment by DanielRBaird** [comment]
Looks like the trick is to specify both Xcode 12.5.1 and macOS 11.

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-VERSION=0.7.2
+VERSION=0.7.3
```
Comment (benasher44) [inline gradle.properties:1]: Should prob be 0.8

**Review by benasher44 — APPROVED** [review]

File: gradle.properties (hunk 1 above)
Comment (DanielRBaird) [reply gradle.properties:1]: Yeah, good point. I'll bump to 0.8

**Review by benasher44 — APPROVED** [review]
don't forget README update as well!

**Comment by DanielRBaird** [comment]
> don't forget README update as well!

Yep, I'll update when I do the release. Probably won't get around to that til tomorrow.

**Pull request merged**